examples/incredible-dot-product/src/config/dot-operator.toml

```
@@ -1,4 +1,4 @@
-bls_private_key = "1371012690269088913462269866874713266643928125698382731338806296762673180359922"
+bls_signer_config.private_key = "1371012690269088913462269866874713266643928125698382731338806296762673180359922"
```

Let's rename `bls_signer_config` to `bls_signer`, so it ends up like:

```suggestion
bls_signer.private_key = "1371012690269088913462269866874713266643928125698382731338806296762673180359922"
```

---

Done [c0f263c](https://github.com/Layr-Labs/eigensdk-rs/pull/544/commits/c0f263c1f8c076010c730d7f6c40dfec2732d8a3)